content/djangogirls.html

```diff
@@ -18,5 +18,7 @@
 with empathy.</p>
 
 <p>To register your interest in attending please
-<a href="http://djangogirls.org/coventry/apply/">fill in this form</a>.</p>
+<a href="http://djangogirls.org/coventry/apply/">fill in this form</a>.
+<a href="http://blog.djangogirls.org/post/91067112853/djangogirls-how-we-scored-applications">This blog post</a>
```

Comment:
Can you please use better link text for a11y? https://twitter.com/NeilMilliken/status/610568082382262274